Makefile

```diff
@@ -558,8 +557,8 @@ test: localunit localintegration remoteintegration localsystem remotesystem  ##
 
 .PHONY: ginkgo-run
 ginkgo-run: .install.ginkgo
-	ACK_GINKGO_RC=true $(GINKGO) version
-	ACK_GINKGO_RC=true $(GINKGO) -v $(TESTFLAGS) -tags "$(TAGS) remote" $(GINKGOTIMEOUT) -cover -flakeAttempts 3 -progress -trace -noColor -nodes $(GINKGONODES) -debug $(GINKGOWHAT) $(HACK)
```

Comment:
Removing `-cover` looks correct to me: it's handled in the `localunit` target above. I've traced the presence of `-cover` in this line all the way back to 2018, and believe it to be a copy/paste from the 2017 CRI-O import. But since there's a slight possibility that I'm wrong, I'm calling attention to it anyway. If anyone sees a reason to keep this, please speak now.